qdyn/output.f90

```diff
@@ -302,7 +315,9 @@ subroutine ot_init(pb)
   pb%ot%v_pre2 = 0.d0
 
   ! Number of ot output quantities
-  pb%ot%not = 11
+  ! pb%ot%not = 11
```

Comment:
Is it correct that 1 quantity was removed, and 2 were added (going from `pb%ot%not = 11` to `pb%ot%not = 12`)? If yes, you can remove the traces of the old code.

Reply:
yes, exactly. I was keeping for reminder purposes. removed.